R/tm_a_pca.R

```diff
@@ -290,26 +320,22 @@ srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, pl
     computation <- reactive({
       req(merged$anl_q_r())
       # inputs
-      keep_cols <- as.character(merged$anl_input_r()$columns_source$dat)
+      keep_cols <- selector_list()$dat()$select
```

Comment:
as discussed, we should change these back to merged$anl_input_r() throughout -> and maybe we should add an issue into teal.transform to give us helper functions to access these things